hamcrest/Hamcrest/Core/HasToString.php

```diff
@@ -10,7 +10,7 @@
 use Hamcrest\Util;
 
 /**
- * Matches if array size satisfies a nested matcher.
+ * Matches an examined object whose toString method returns a value that satisfies the specified matcher.
```

Comment:
Please use text from the [relevant java file](https://github.com/hamcrest/JavaHamcrest/blob/master/hamcrest/src/main/java/org/hamcrest/object/HasToString.java) to preserve formatting.

The above-proposed text doesn't the matching Java file text.

P.S.
The previous text looks like a copy/paste from the https://github.com/hamcrest/JavaHamcrest/blob/master/hamcrest/src/main/java/org/hamcrest/collection/IsArrayWithSize.java file.